jplag/src/main/java/de/jplag/reportingV2/JsonReport.java

```diff
@@ -0,0 +1,29 @@
+package de.jplag.reportingV2;
+
+import java.util.List;
+
+import de.jplag.JPlagResult;
+import de.jplag.reportingV2.jsonfactory.JsonFactory;
+import de.jplag.reportingV2.reportobject.ReportObjectFactory;
+import de.jplag.reportingV2.reportobject.model.JPlagReport;
+
+// ReportImplementation -> JsonReport
+
+/**
+ * A report generator which reports the JPlagResult in Json format.
+ */
+public class JsonReport implements Report {
```

Comment:
Currently, this report is not used by Jplag? In the course of replacing the old reporting code you could also insert the new reference in the main class